src/main/java/dan200/computercraft/core/apis/http/request/HttpRequestHandler.java

```diff
@@ -62,6 +62,28 @@
         this.options = options;
     }
 
+    private String getHostHeader()
+    {
+        String host = uri.getHost();
+
+        if( uri.getPort() >= 0 )
+        {
+            int defaultPort = -1;
+            switch ( uri.getScheme() )
```

Comment:
I'd be inclined just to replace this with `uri.getScheme().equalsIgnoreCase("https") ? 443 : 80` - it's what we do in `HttpRequest`.

Alternatively, just drop the default port check. From what I can tell reading the spec, this _should_ be fine?? It'll only get triggered if people explicitly provide a port, so it doesn't seem like the end of the world if we include it.

Reply:
Although okhttp and aiohttp.client do cut off default ports, I agree on dropping this chek.